x-pack/plugins/observability_solution/apm/server/lib/helpers/create_es_client/create_assets_es_client/create_assets_es_clients.ts

```diff
@@ -15,8 +17,14 @@ import {
 import { withApmSpan } from '../../../../utils/with_apm_span';
 import { EntityType } from '../../../../routes/entities/types';
 
-const ENTITIES_LATEST_INDEX_NAME = `entities-${EntityType.SERVICE}-latest`;
-const ENTITIES_HISTORY_INDEX_NAME = `entities-${EntityType.SERVICE}-history`;
+const ENTITIES_LATEST_INDEX_NAME = entitiesAliasPattern({
```

Comment:
nit is the naming off ? maybe `SERVICE_ENTITIES_LATEST_ALIAS`

Reply:
Yep, I fixed this in the last PR, let me fix it here too

Reply:
Fixed in https://github.com/elastic/kibana/pull/190142/commits/97a02909dd449e357558e839744d6a8af57b8129